docs/_config.yml

```diff
@@ -25,4 +25,4 @@ exclude: [_docs]
 baseurl: http://samza.apache.org
 version: latest
 # this is the version you will go if you click 'switch version' in "latest" pages.
-latest-release: '0.13'
+latest-release: '0.13.1'
```

Comment:
No need to change this. We only track major versions in doc site (i.e. 0.13.1 doc will over write 0.13 version doc, not creating a new 0.13.1 version doc site).